### PR TITLE
test: Enable setroubleshoot on RHEL

### PR DESCRIPTION
**Pull request by dperpeet**
Install setroubleshoot-server on rhel systems if available.
Also, don't skip the tests categorically on rhel - check if the system is new enough, first.

### CODE REVIEW

File: test/images/scripts/rhel-7.setup [hunk 1]
```diff
@@ -84,6 +84,15 @@ sos \
 tuned \
 "
 
+# If the RHEL version is new enough, install setroubleshoot-server
```
Comment (petervo) [inline test/images/scripts/rhel-7.setup]: Wouldn't we just change this once our main image is new enough? It's not like we'll need to go back and create older images again.

**Comment by stefwalter** [comment]
tools/cockpit.spec needs an update.

File: test/images/scripts/rhel-7.setup (hunk 1 above)
Comment (dperpeet) [inline test/images/scripts/rhel-7.setup]: > Wouldn't we just change this once our main image is new enough? It's not like we'll need to go back and create older images again.

The problem is that for quite a while, we'll be using old and new images in parallel - i.e. QE will use early 7.3 composes and want to test those, while upstream cockpit still uses 7.2.

**Comment by dperpeet** [comment]
> tools/cockpit.spec needs an update.

Indeed it does, if we want to use setroubleshoot on RHEL. For now, I would like to only enable the tests. We can update cockpit.spec once we have the packages available to actually test the newer version upstream. If I'm not mistaken, we discussed folding the troubleshooting code into cockpit-shell for now (on rhel/centos).

**Comment by stefwalter** [comment]
Wasn't the idea that selinux troulbeshooting would work on RHEL once 7.3 was released?

**Comment by stefwalter** [comment]
I'm rebuilding the rhel-7 image to do this

**Comment by stefwalter** [comment]
Replaced by #4977

**Pull request closed without merging**